### PR TITLE
[FE] Shifting the Default-Setting Chrome Extension from Search-First to Chat-Only

### PR DESCRIPTION
**Pull request by ClintMWI**
[[FE] Shifting the Default-Setting Chrome Extension from Search-First to Chat-Only](https://app.asana.com/0/1203136326293515/1207234439263959/f)

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,63 +1,29 @@
...
-        }
-      ]
-    },
-  "permissions": [
-    "declarativeNetRequestWithHostAccess", "storage"
-  ], 
-  "host_permissions": [
-  "*://*/*"
   ],
+  "permissions": ["declarativeNetRequestWithHostAccess", "storage"],
```
Comment (davidli3100) [inline public/manifest.json:12]: i don't think we need declarativeNetRequest as a perm anymore since we removed those rules

File: src/App.tsx
Comment (davidli3100) [inline src/App.tsx:1]: can we just entirely delete this?

File: src/index.tsx
Comment (davidli3100) [inline src/index.tsx:1]: do we need this too?

Comment (ClintMWI) [reply src/index.tsx:1]: I'm pretty sure that we need both of these still. If I delete them, it won't build.

File: src/App.tsx
Comment (ClintMWI) [reply src/App.tsx:1]: I think this contains what updates the settings in chrome.

**Review by davidli3100 — APPROVED** [review]

**Pull request merged**